index.js

```diff
@@ -1080,6 +1122,7 @@
             ignoreFirstPageLoad: ignoreFirstPageLoad,
             logAllCalls: logAllCalls,
             pageEvent: pageEvent,
+            readFromRoute = readFromRoute,
```

Comment:
This needs to be a property declaration: `readFromRoute: readFromRoute,` to compile.

Make sure to run the grunt tests as the linter will catch anything off like this.